backend/src/handlers/hike.ts

```diff
@@ -0,0 +1,72 @@
+import { Request, Response } from "express";
+import Hike from "../models/hike.model";
+
+export const handleaddhike = async (req: Request, res: Response) => {
+  try {
+    const hike = await Hike.create({
+      img: req.body.img,
+      title: req.body.title,
+      text: req.body.text,
```

Comment:
Same here, pass only needed arguments to the handler. Please speak with Lior about this :)